agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+    4. During your employment, you may be given permission to copy take copies of documents, including computer programs, planning documents and ancillary documentation, in which copyright subsists. Copies may be made available to you in different way — for example by giving you access to a version control repository. Unless we agree otherwise, your licence to retain and use copies of any such material is limited to the period of your employment with us. Once you have ceased to work with us you agree that you will delete all copies of any such material (including any backup copies) that remain in your possession or control.
+
+16. ### Loyalty
+    1. You agree that while you are employed by us you will not, without our prior written consent, carry out any work for or take employment with any employer who is or could reasonably become a competitor of our company.
+    2. Similarly, you agree not to carry out any work for any employer who is or could reasonably become a client of our company, other than the work you do as part of your employment with us.
+    3. You may carry out any other work provided it does not interfere with the performance of your duties working for us.
+
+17. ### Gratuituies
+    1. You must immediately report in writing any offer of gifts or hospitality by a customer, supplier or any other such person with an interest in or connection to the company if it might reasonably be seen to compromise your judgement or integity.
+    2. You must not accept any such offer without the prior agreement of the company.
+    4. You must not offer any gifts or hospitality to a customer, supplier or any other such person with an interest in or connection to the company if it might reasonably be seen to compromise their judgement or integity.
+    3. "Gifts" and "hospitality" do not include ordinary, day-to-day items routinely purchased for others, such as refreshments in meetings.
+
+18. ### General duties
+    1. During your employment with us, you must:
+        1. complete your work expeditiously, dilligently, and to the best of your ability;
+        2. as far as you are able, behave in ways which embody dxw's values, as published in the Playbook;
+        3. follow dxw's policies, as published and varied from time to time.
+
+19. ### Disciplinary and grievance procedures
+    1. Details of dxw's discplinary and grievance procedures are published in the Playbook.
+
+20. ### Playbook
+    1. Further rules concerning your employment with us are contained in the _Working arrangements_ section of the Playbook, which forms a part of your contract of employment.
+    2. The _Working arrangements_ section of the Playbook will be varied by us from time to time. We will give you notice of any such changes, which will be incorporated into your contract of employment from the date on which they are notified to you.
```

Comment:
Again: division between what's in the playbook vs contract. 

If this was code I'd say that the abstraction is wrong here/the seam between these two classes isn't in the right place/there's too much coupling/this document suffers from the '[Feature Envy](https://sourcemaking.com/refactoring/smells/feature-envy)' code smell.